packages/block-library/src/code/theme.scss

```diff
@@ -1,8 +1,8 @@
 .wp-block-code {
 	font-family: $editor-html-font;
-	font-size: $text-editor-font-size;
+	font-size: 0.9em;
 	color: $gray-900;
-	padding: 0.8em 1em;
+	padding: 0.9em 1.1em;
```

Comment:
As far as I can tell, there's no reason to change this, since it was already relative to the current `font-size`?

Reply:
You are correct, there is no reason to change this one.
Changed it back to its previous values. :+1: